### PR TITLE
Added Web Server category

### PR DESCRIPTION
**Pull request by AshikNesin**
I have added a new category called **Web Server** with [Harp](https://github.com/cheeriojs/cheerio), [Live Server](https://github.com/tapio/live-server) in it and moved [http-server](https://github.com/nodeapps/http-server) from **Command-line apps** to it.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -616,6 +615,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [cool-ascii-faces](https://github.com/maxogden/cool-ascii-faces) - Get some cool ascii faces.
 - [cat-ascii-faces](https://github.com/melaniecebula/cat-ascii-faces) - ₍˄·͈༝·͈˄₎◞ ̑̑ෆ⃛ (=ↀωↀ=)✧ (^･o･^)ﾉ”
 
+### Web Server
+
+- [http-server](https://github.com/nodeapps/http-server) - Simple, zero-config command-line HTTP server.
+- [Harp](https://github.com/cheeriojs/cheerio) - Static web server with built-in preprocessing like Markdown, Jade, EJS, LESS, Stylus, Sass and CoffeeScript.
```
Comment (shinnn) [inline readme.md]: `Static web server with built-in preprocessing.` or reduce the words after `like` to 2 or 3.

And I doubt whether we can regard a server including pre-processing function as a "static server", though I don't understand the term "static server" well.

Comment (shinnn) [inline readme.md]: https://github.com/cheeriojs/cheerio → https://github.com/sintaxi/harp

**Comment by shinnn** [comment]
IMHO harp is not suitable for this awesome-nodejs list. I personally don't like the idea of built-in preprocessor (except for the standard-compatible ones like [babel](https://babeljs.io/) and [cssnext](http://cssnext.io/)). They often just bloat up the package.

**Comment by AshikNesin** [comment]
@shinnn It might be useful to some that's why I added it. Do I need to remove it?

File: readme.md (hunk 1 above)
Comment (shinnn) [inline readme.md]: > I doubt whether we can regard a server including pre-processing function as a "static server", though I don't understand the term "static server" very well.

Do you have any idea? @sindresorhus @arthurvr @kevva @floatdrop

**Comment by shinnn** [comment]
> Do I need to remove it?

Please wait for another opinion.

**Comment by sindresorhus** [comment]
Not interested in Harp or a separate category.

**Pull request closed without merging**